src/globus_cli/commands/flows/create.py

```diff
@@ -0,0 +1,196 @@
+from __future__ import annotations
+
+import typing as t
+
+import click
+
+from globus_cli.login_manager import LoginManager
+from globus_cli.parsing import JSONStringOrFile, command
+from globus_cli.termio import Field, TextMode, display, formatters
+
+ROLE_TYPES = ("flow_viewer", "flow_starter", "flow_administrator", "flow_owner")
+
+
+@command("create", short_help="Create a flow")
+@click.argument(
+    "title",
+    type=str,
+)
+@click.argument(
+    "definition",
+    type=JSONStringOrFile(),
+    metavar="DEFINITION",
+)
+@click.option(
+    "--input-schema",
+    "input_schema",
+    type=JSONStringOrFile(),
+    help="""
+        The JSON input schema that governs the parameters
+        used to start the flow.
+
+        The input document may be specified inline,
+        or it may be a path to a JSON file, prefixed with "file:".
+
+        Example: Inline JSON:
+
+        \b
+            --input_schema '{"properties": {"src": {"type": "string"}}}'
+
+        Example: Path to JSON file:
+
+        \b
+            --input_schema file:schema.json
```

Comment:
_Assuming_ we take the JSONStringOrFile rewrite, I'd change this to `--input-schema schema.json` to bias us away from the `file:` prefix syntax.

I also notice `--input_schema` here and above, which need minor correction.

Reply:
Totally missed the underscore-vs-hyphen. Good catch!